src/authentication.ts

```diff
@@ -81,7 +81,6 @@ export const fetchTokens = (config: TInternalConfig): Promise<TTokenResponse> =>
   const tokenRequest: TTokenRequestWithCodeAndVerifier = {
```

Comment:
Should update the type here as well

Reply:
https://github.com/soofstad/react-oauth2-pkce/blob/a23e7081a7ce85c1b5a16cf310a2bb0e6fd4a0b6/src/Types.ts#L3-L13

Reply:
Yes. Good catch. I'll update it.